### PR TITLE
Branch0101

### PR DESCRIPTION
**Pull request by J1ers**

### CODE REVIEW

File: builder/core/src/main/java/com/antgroup/openspg/builder/core/strategy/linking/impl/SearchBasedLinking.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    List<BaseQuery> queries = new ArrayList<>(4);
+    queries.add(new MatchQuery("id", rawValue));
+    queries.add(new MatchQuery("name", rawValue));
+    if (objectTypeRef.isConceptType()) {
+      queries.add(new MatchQuery("alias", rawValue));
+      queries.add(new MatchQuery("stdId", rawValue));
+    }
+    QueryGroup queryGroup = new QueryGroup(queries, OperatorType.OR);
+    request.setQuery(queryGroup);
+    request.setSize(50);
```
Comment (leywar) [inline builder/core/src/main/java/com/antgroup/openspg/builder/core/strategy/linking/impl/SearchBasedLinking.java:69]: add global variable for search engine truncation length setting.

File: builder/runner/local/src/main/java/com/antgroup/openspg/builder/runner/local/LocalBuilderRunner.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    sourceReader.init(context);
+    sinkWriter =
+        logicalPlan.sinkNodes().stream().map(SinkWriterFactory::getSinkWriter).findFirst().get();
+    sinkWriter.init(context);
+
+    PhysicalPlan physicalPlan = PhysicalPlan.plan(logicalPlan);
+    builderExecutor = new DefaultBuilderExecutor();
+    builderExecutor.init(physicalPlan, context);
+
+    // 构建指标统计，并将构建指标输出到log
```
Comment (leywar) [inline builder/runner/local/src/main/java/com/antgroup/openspg/builder/runner/local/LocalBuilderRunner.java:69]: Please write comments in English.

File: builder/runner/local/src/main/java/com/antgroup/openspg/builder/runner/local/LocalBuilderRunner.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    builderMetric.reportToLog();
+
+    if (context.isEnableLeadTo()) {
+      reasonProcessor = new ReasonProcessor();
+      reasonProcessor.init(context);
+    }
+  }
+
+  @Override
+  public void execute() throws Exception {
```
Comment (leywar) [inline builder/runner/local/src/main/java/com/antgroup/openspg/builder/runner/local/LocalBuilderRunner.java:80]: add comments to describe the `execute` process in detail

**Review by leywar — CHANGES_REQUESTED** [review]

File: reasoner/common/src/main/java/com/antgroup/openspg/reasoner/common/graph/vertex/impl/VertexId.java [hunk 4]
```diff
@@ -33,13 +34,16 @@ public class VertexId implements IVertexId {
    */
   private final byte[] vertexIdBytes;
 
+  @Getter private String bizId;
```
Comment (youdonghai) [inline reasoner/common/src/main/java/com/antgroup/openspg/reasoner/common/graph/vertex/impl/VertexId.java:37]: We cannot add the bizId here as this class represents a segment of binary data used as a fixed-length key for RocksDB. Adding the bizId here would result in data from RocksDBGraphState being unable to deserialize the bizId, rendering the entire engine system inoperable.

**Review by youdonghai — CHANGES_REQUESTED** [review]

**Pull request closed without merging**